python/main.py

```diff
@@ -67,11 +70,12 @@ def get_item(item_id):
         ON category.id = items.category_id 
         WHERE items.id = (?)
     ''', (item_id, ))
-    logger.info(f"Get item of id:")
+    logger.info(f"Returning the item of id: {itemid}.")
```

Comment:
I think there is a typo. It's {item_id}, right?

Reply:
Opps! Thanks for pointing that out!
I have corrected the typo in [this commit](https://github.com/tsoibet/mercari-build-training-2022/pull/14/commits/b6c0d080b4fb465bf8c2ed5407a2b75fc17697ce).
Could you please check again?